scipy/stats/stats.py

```diff
@@ -663,9 +663,17 @@ def tmin(a, lowerlimit=None, axis=0, inclusive=True, nan_policy='propagate'):
         This flag determines whether values exactly equal to the lower limit
         are included.  The default value is True.
     nan_policy : {'propagate', 'raise', 'omit'}, optional
-        - 'propagate': returns nan for 1-D array. For 2-D array, if
-           nan is present at `m, n`, nan is returned in the result ndarray
-           at the position of `n` for `axis=0` or `m` for `axis=1`
+        - 'propagate': ndarray of dimension `N-1` is returned for the input
+          ndarray of dimension `N`. The dimension deduction is due to that
+          the computation axis is taken away.
+          Given the computation axis `axis=n`, if any nan is found in the
+          hyperplane (`x_1=y_1, x_2=y_2,..., x_n-1=y_n-1,
+          x_n+1=y_n+1,..., x_N=y_N`), nan is placed at
+          `(y_1, y_2,...,y_n-1, y_n+1,..., y_N)` in the result ndarray.
```

Comment:
While this is technically correct when `axis` is an integer I don't think it makes the effect any clearer. Maybe a simple "where the calculation is affected by a nan value" is sufficient?

Reply:
I agree with @Kai-Striega that the `nan_policy` documentation doesn't need to be so explicit. The documentation of the return value doesn't go into this level of detail, so I think it would be inconsistent to be so detailed here.

Maybe there's a compromise between @Kai-Striega 's suggested wording and what is here. There is some precedent to using the term "axis slice" to refer to the slice of the array (like a column, row, etc...) that is used for calculating a quantity, and I'm assuming that it is understood that the quantity at each entry of the returned array is a "statistic":

 > Propagates nans to the output: if a nan is present in the axis slice along which the statistic is computed, the output entry corresponding with the statistic will be nan.